test/integration/default/inspec/default_spec.rb

```diff
@@ -2,3 +2,22 @@
 
 # The InSpec reference, with examples and extensive documentation, can be
 # found at https://docs.chef.io/inspec/resources/
+
+describe user 'seagl' do
+  it { should exist }
+  its('home') { should cmp '/home/seagl' }
+end
+
+describe command('sudo -u seagl sudo -l') do
```

Comment:
This is a better way :)
```suggestion
describe command('sudo -U seagl -l') do
```